### PR TITLE
user backend/authpage.js/signupform.js completed

### PR DESCRIPTION
**Pull request by Jcasanova1990**
D       controllers/api/todos.js
D       models/todo.js
D       routes/api/todos.js
D       src/components/Todo/Todo.js
D       src/components/Todo/Todo.module.scss
D       src/components/TodoList/TodoList.js
D       src/components/TodoList/TodoList.module.scss

C user model
C user routes
C user controller
C signup page
C authpage

### CODE REVIEW

**Comment by Jcasanova1990** [comment]
it just requires a front end for this page

File: src/pages/AuthPage/AuthPage.js
Comment (dwheeler7) [inline src/pages/AuthPage/AuthPage.js:1]: Tyler already made an AuthPage component, please delete.

File: routes/api/todos.js
Comment (dwheeler7) [inline routes/api/todos.js:1]: Deleting todos routes broke the app. Delete the middleware in app-server.js referencing todos.

File: routes/api/users.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const router = express.Router()
+const {dataContoller, apiController} =require('../../controllers/api/users')
+const ensureLoggedIn = require('../../config/ensureLoggedIn')
```
Comment (dwheeler7) [inline routes/api/users.js:4]: This file doesn't exist.

**Review by dwheeler7 — COMMENTED** [review]
Please make changes i put in review.

File: routes/api/users.js (hunk 1 above)
Comment (Jcasanova1990) [reply routes/api/users.js:4]: this was created

**Pull request closed without merging**